src/test/java/org/example/commands/AddDependencyCommandTest.java

```diff
@@ -18,9 +17,10 @@
 class AddDependencyCommandTest {
```

Comment:
divide the pr based on functionality. Testcases and exception handling should go hand in hand with logic writing. you can create 5 pr for add, delete, getchild etc. 
If u change same thing in next pr there is no point reviewing this pr

Reply:
I will remember this from the next assignment and will follow all these guidelines